index.mjs

```diff
@@ -90,38 +89,39 @@ const sign_and_request = async({ sign, host, url, body }) => {
 		body: body_string,
 	})
 
-	const [ failed_response, successful_response ] = await catchify(post(url, {
-		headers: {
-			...headers,
-			authorization,
-		},
-		body: body_string,
-	}))
+	let successful_response
+	try {
+		successful_response = await post(url, {
```

Comment:
this makes me want to put `catchify` back 😂

Reply:
done... it improved this commit: a34b6ca66e4f7b069109899474070b2ed560728c
and probably the one after